interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -550,7 +550,7 @@ public function getMedications($pid, $encounter)
 
     public function getProblemList($pid, $encounter)
     {
-        (new UuidRegistry(['table_name' => 'lists']))->createMissingUuids();
+        UuidRegistry::createMissingUuidsForTables(['lists']);
```

Comment:
I couldn't get this to work! strange... I'm going to add back in my version too be sure.

Reply:
What error were you hitting?

Reply:
This works and nope unsure why. Maybe updated environment.